collector/collector.go

```diff
@@ -19,6 +19,7 @@ func NewCollector(db Db, url string) *eventsCollector {
 	return &eventsCollector{url, make(chan struct{}), db}
 }
 
+// Db defines which events should be stores by any db that the collector uses
 type Db interface {
```

Comment:
shouldn't this be `DB` ?

Reply:
I agree with @y0sher here. While the linter only catches `Id`, the guidelines are that acronyms should be in the same case (either lower or upper).

I had a case where I didn't follow this advice: `poetDbApi` - I made it `poetDbAPI` because I find `poetDBAPI` to be harder to read (but I'm open to changing it if others disagree with my call).